### PR TITLE
UIIN-2179: Browse contributors | Show an indicator that contributor value is controlled/linked to an authority record

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
## Purpose
Add a link to the MARC authority record when an `Instance` record is linked.

## Approach
Fake accessible link created using a span to avoid `Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>`

## Issues
[UIIN-2179](https://issues.folio.org/browse/UIIN-2179)

## Screencast

https://user-images.githubusercontent.com/77053927/192547961-94ed22ac-b0ca-4afd-a1eb-3ac23cbe8002.mp4

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/InstancesList/InstancesList.js [hunk 1]
```diff
@@ -971,7 +979,49 @@ class InstancesList extends React.Component {
...
+              <>
+                <Tooltip
+                  id="marc-authority-tooltip"
+                  text={intl.formatMessage({ id: 'ui-inventory.linkedToMarcAuthority' })}
+                >
+                  {({ ref, ariaIds }) => {
+                    const url = `marc-authorities/authorities/${r.authorityId}?segment=search`;
+
+                    return (
+                      <span
```
Comment (BogdanDenis) [inline src/components/InstancesList/InstancesList.js:996]: Can we use a `<TextLink>` from `stripes-components` here?

File: src/components/InstancesList/InstancesList.js [hunk 2]
```diff
@@ -971,7 +979,49 @@ class InstancesList extends React.Component {
       },
       'contributor': r => {
         if (r?.totalRecords) {
-          return getFullMatchRecord(r?.name, r.isAnchor);
+          const fullMuchRecord = getFullMatchRecord(r.name, r.isAnchor);
```
Comment (BogdanDenis) [inline src/components/InstancesList/InstancesList.js:982]: Typo here, should be `fullMatchRecord`

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/InstancesList/InstancesList.js (hunk 1 above)
Comment (BogdanDenis) [reply src/components/InstancesList/InstancesList.js:996]: Oh, actually, `<IconButton>` has a `to` prop that turns in into a link. Perhaps we can use that too?

File: src/components/InstancesList/InstancesList.js (hunk 2 above)
Comment (Dmytro-Melnyshyn) [reply src/components/InstancesList/InstancesList.js:982]: Thanks for noticing.

File: src/components/InstancesList/InstancesList.js (hunk 1 above)
Comment (BogdanDenis) [reply src/components/InstancesList/InstancesList.js:996]: @Dmytro-Melnyshyn oh because a row is a `<a>` element itself, got it. Then it's ok

**Review by BogdanDenis — APPROVED** [review]

**Review by mkuklis — APPROVED** [review]

**Pull request merged**